android/java/org/chromium/chrome/browser/crypto_wallet/util/AndroidUtils.java

```diff
@@ -98,4 +99,14 @@ private static void setViewVisibility(boolean isVisible, View... views) {
     public static int getScreenHeight() {
         return Resources.getSystem().getDisplayMetrics().heightPixels;
     }
+
+    /**
+     * Check if the fragment is safe to update its UI
+     * @param frag instance
+     * @return true if Fragment UI can be updated otherwise false
+     */
+    public static boolean canUpdateFragmentUi(Fragment frag) {
+        return !(frag.isRemoving() || frag.getActivity() == null || frag.isDetached()
```

Comment:
It's only for fragment itself not for the container or parent.

Reply:
The function name itself explains what it does and I already added javadoc for you specially that should be enough. No need to add info about `isRemovingParent` as it is not related and needed. Would suggest to add minor or subjective review as nits because there are often many than the crucial ones.